### DISCUSSION TITLE
Allow/Prevent Email Notification based on submission content

### DISCUSSION THREAD

**Discussion by icreatestuff** [solspace/craft-freeform#672] [Questions & Answers]
Is it possible to prevent an automatic email notification (submitter, or admin notifications for example) from being sent unless the submission meets some specific criteria. For example only send the email if:

- Status is 'Closed'
- Checkbox field is checked.

I couldn't see anything in the [email notifications docs](https://docs.solspace.com/craft/freeform/v4/overview/email-notifications/) to suggest this was possible yet.

Thanks

**Comment by kjmartens** [comment]
What you could do is use Twig in the Admin Notify area to create conditionals based on the submission data to handle emails being sent:
https://docs.solspace.com/craft/freeform/v4/overview/email-notifications/#admin-notifications

``` twig
{% if 'option1' in myCheckboxGroupField.value %}
bigbob@acmewidgets.net 
{% endif %}
{% if 'option2' in myCheckboxGroupField.value %}
fuzzypants@acmewidgets.net 
{% endif %}
{% if 'option3' in myCheckboxGroupField.value %}
hotsauce@acmewidgets.net 
{% endif %}
```

**Reply by icreatestuff** [reply]
Hi @kjmartens thanks for the help.

The email in question is for a user/submitter notification so I don't _think_ what your proposing will work.

In an ideal world we'd be able to trigger an email notification to the submitter when the submission status changes e.g. from 'Pending' to 'Closed' - a bit like is being requested [here](https://github.com/solspace/craft-freeform/discussions/120). I don't believe that is yet an option so I was looking for another way to achieve something similar.

I'd be more than happy to trigger an email notification via one of Freeform's service classes/APIs within a custom module or plugin - is that possible?

**Reply by kjmartens** [reply]
Unfortunately there currently are no events that allow you to handle the status change. After checking with a developer, they suggest it _might_ be possible by using the https://docs.solspace.com/craft/freeform/v4/developer/events/submission/#after-a-submission-is-submitted event and compare the previous status with the current one. However, you would have to make your own email notification that has to be sent and use the mailer service to send it if you go this route. 🙂
